lib/AST/ConformanceLookup.cpp

```diff
@@ -769,7 +769,7 @@ bool TypeBase::isSendableType() {
     return fas->isSendable();
 
   auto conformance = proto->getParentModule()->checkConformance(this, proto);
-  if (conformance.isInvalid())
+  if (conformance.isInvalid() || conformance.hasUnavailableConformance())
     return false;
 
   // Look for missing Sendable conformances.
```

Comment:
The `conformance.forEachMissingConformance(...)` thing is equivalent to `conformance.hasMissingConformances()`. However, you can remove that check entirely by passing `/*allowMissing=*/false` to the checkConformance() call on the previous line.

Reply:
I'll do that in a follow on.